packages/progress-bar/src/ProgressBar.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Heading } from "@kaizen/component-library"
+import React from "react"
+import classnames from "classnames"
+import styles from "./ProgressBar.scss"
+
+type Props = {
+  value: Value
+  variant: Variant
+  mood: Mood
+  subtext?: string
+  labelExtraContent?: React.ReactNode
+}
+
+type Value =
+  | { kind: "percentage"; value: number }
+  | { kind: "fraction"; value: number; max: number }
+
+type Mood = "positive" | "informative" | "negative" | "cautionary"
+
+type Variant = "static" | "loading"
+
+const progressClassNames = (props: Props) => {
+  const { mood } = props
+  return classnames({
+    [styles.positive]: mood === "positive",
+    [styles.informative]: mood === "informative",
+    [styles.cautionary]: mood === "cautionary",
+    [styles.negative]: mood === "negative",
+    [styles.animating]: isAnimating(props),
+  })
+}
+
+function calculatePercentage(value: Value) {
+  return value.kind === "percentage"
+    ? value.value
+    : (value.value / value.max) * 100.0
+}
+
+function isAnimating({ variant, mood, value }: Props) {
+  if (variant === "static") {
+    return false
+  }
+  if (mood === "negative") {
```

Comment:
This came from the legacy component. If it's in an error state, it's not going to be progressing, so it makes sense to stop the animation (sounds good to me)

Reply:
I don't believe we should block animations for consumers that want negative moods. I'd imagine they were called `moods` in part so that they were not necessarily associated with error states? I may be progressing to something bad and gosh darn it I want to animate while I get there! Can animation be its own prop instead?